src/js/Components/LottoNumForm.js

```diff
@@ -0,0 +1,72 @@
+import Component from "../Core/Component.js";
+import LottoPurchasedNum from "./LottoPurchasedNum.js";
+import { SHOW_NUMBERS } from "../modules/actions.js";
+import { showNumbers } from "../modules/creator.js";
+import { store } from "../modules/store.js";
+import { $ } from "../utils/dom.js";
+
+const actionMap = {
+  SHOW_NUMBERS: (isShow) => {
+    store.dispatch(showNumbers(isShow));
+  },
+};
+
+export default class LottoNumForm extends Component {
+  constructor(target) {
+    super(target);
+  }
+
+  setEvent(target) {
+    this.buttonEvent(target);
+    this.toggleEvent();
+  }
+
+  buttonEvent() {}
+  toggleEvent() {
+    $("#lotto-switch").addEventListener("click", (event) => {
+      actionMap[SHOW_NUMBERS](event.target.checked);
+      const props = store.getState();
+      console.log(props);
+      console.log(event.target.checked);
+    });
+  }
+
+  template() {
+    const props = store.getState();
```

Comment:
```suggestion
    const {price, isShow} = store.getState();
```
구조 분해 할당을 활용해도 좋을꺼 같아요